src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/integration_test_base.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from unittest import TestCase, skipIf
+from datetime import datetime
+from azure.storage.blob.baseblobservice import BaseBlobService
+
+
+@skipIf(os.environ.get('Travis', 'false') == 'true', 'Integration tests are skipped in Travis CI')
+class StorageIntegrationTestBase(TestCase):
```

Comment:
Why do we do this instead of a VCR test recording? Is this essentially a local "run live" test?